cmd/command.go

```diff
@@ -223,6 +223,39 @@ func rpop(cli *Client, args [][]byte) (interface{}, error) {
 	return popInternal(cli.db, args, false)
 }
 
+func lmove(cli *Client, args [][]byte) (interface{}, error) {
+	if len(args) != 4 {
+		return nil, newWrongNumOfArgsError("lmove")
+	}
+
+	srcKey, dstKey := args[0], args[1]
+	from, to := strings.ToLower(string(args[2])), strings.ToLower(string(args[3]))
+	var srcIsLeft, dstIsLeft bool
+
+	if from == "left" {
+		srcIsLeft = true
+	} else if from == "right" {
+		srcIsLeft = false
+	} else {
+		return nil, errSyntax
+	}
+
+	if to == "left" {
+		dstIsLeft = true
+	} else if to == "right" {
+		dstIsLeft = false
+	} else {
+		return nil, errSyntax
+	}
+
+	value, err := cli.db.LMove(srcKey, dstKey, srcIsLeft, dstIsLeft)
+	if err != nil {
+		return nil, err
+	}
+
+	return value, nil
+}
```

Comment:
I think you can return directly the function call. 
```suggestion
	return cli.db.LMove(srcKey, dstKey, srcIsLeft, dstIsLeft)
}
```